GitUI/Editor/FileViewer.cs

```diff
@@ -197,6 +198,16 @@ public FileViewer()
 
             _fullPathResolver = new FullPathResolver(() => Module.WorkingDir);
             SupportLinePatching = false;
+
+            _NO_TRANSLATE_resetSelectedLinesConfirmationDialog = new()
+            {
+                Text = TranslatedStrings.ResetSelectedLinesConfirmation,
+                Caption = TranslatedStrings.ResetChangesCaption,
+                Icon = TaskDialogIcon.Warning,
+                Buttons = { TaskDialogButton.Yes, TaskDialogButton.No },
+                DefaultButton = TaskDialogButton.Yes,
```

Comment:
Is this a good idea? I _think_ the default button for destructive operations should be the one that doesn't lead to data loss.

Reply:
`Yes` is currently the default in the `MessageBox` and because I use it **a lot** in my daily work (select text, `r`, `enter`, _repeat_) I didn't want to change the behavior even though I agree with what you said.